pkgs/misc/emulators/qmc2/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchurl, qmake, qttools, pkgconfig
+, minizip, zlib
+, qtbase, qtsvg, qtmultimedia, qtwebkit, qttranslations, qtxmlpatterns
+, rsync, SDL2, xwininfo
+, utillinux #colrm
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  name = "qmc2-${version}";
+  version = "0.195";
+
+  src = fetchurl {
+      url = "https://netix.dl.sourceforge.net/project/qmc2/qmc2/${version}/${name}.tar.gz";
+      #url = "mirror://sourceforge/projects/qmc2/files/qmc2/${version}/${name}.tar.gz/download";
```

Comment:
Please delete this comment, anstead of using this `netix...` url use the mirror we have, like this: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/misc/mantisbt.nix#L13

Reply:
problem is that mirror is broken for me, never works and i get their custom 404 ... boring SF.